docs/expressions.rst

```diff
@@ -47,10 +47,26 @@ that order.
 Basic comparison operators
 ==========================
 
+===============================  ========================  =====================
+Operator                         Description               Example
+===============================  ========================  =====================
+``=``, ``==``, ``eq``            Equality                  ``foo == 5``
+``!=``, ``!==``, ``ne``, ``≠``   Inequality                ``bar != 5``
+``>``, ``gt``                    Greater than              ``foo > 5``
+``>=``, ``ge``, ``≥``            Greater than or equal to  ``foo >= 5``
+``<``, ``lt``                    Less than                 ``foo < 5``
+``<=``, ``le``, ``≤``            Less than or equal to     ``foo <= 5``
+``is``                           Identity                  ``foo is True``
+``isnot``                        Inverse identity          ``foobar isnot True``
+===============================  ========================  =====================
+
+
+Boolrule also supports extended comparison operators
+
 =======================  ========================  =====================
```

Comment:
@spjwebster The descriptions look great. However i think you intended to replace the this table.

Reply:
Ah, yes... I was going to document the new ones separately but realised it didn't make too much sense. Thanks